bittensor/errors.py

```diff
@@ -58,6 +58,12 @@ class UnstakeError(ChainTransactionError):
     pass
 
 
+class ChildrenInfoError(ChainTransactionError):
+    """Error raised when a setting a child hotkey transaction fails."""
+
+    pass
```

Comment:
if you have docstring then `pass` isn't required

Reply:
removed this error